tests/mlflow/tensorflow/test_tensorflow_model_export.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import print_function
+
+import os
+import pandas
+import unittest
+
+import numpy as np
+import tensorflow as tf
+import sklearn.datasets as datasets
+
+from mlflow import tensorflow, pyfunc
+from mlflow import tracking
+from mlflow.utils.file_utils import TempDir
+
+
+class TestModelExport(unittest.TestCase):
```

Comment:
Based on conversation with @andrewmchen & @tomasatdatabricks we should refactor this to to pytest format, though IMO we can wait till after we release TF functionality.

Some notes on how this could be done:
* Use the [tmpdir fixture](https://docs.pytest.org/en/latest/tmpdir.html) to generate a temporary test dir.
* See https://github.com/databricks/mlflow/blob/master/tests/tracking/test_tracking.py for an example of pytest-style tests (except I didn't / should have used the tmpdir fixture when writing them :P). 
* Instance variables used in the test method (`self._dnn`, `self._dnn_predict`, `self._X`, `self._feature_names`) can be passed as fixtures. We could have a utility method that returns a tuple of these values & use [pytest.mark.parametrize](https://docs.pytest.org/en/latest/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions) to feed values into the test function. Speaking of which, we should be able to implement the test with just `estimator`, `X`, and `y` fixture values (fit the estimator on X and y, compare predictions on y before & after saving/loading the model).